README.md

```diff
@@ -68,3 +68,9 @@ jenkins/jenkinsnotify.pl syntax OK
 jenkins/openstack-unittest-testconfig.pl syntax OK
 jenkins/track-upstream-and-package.pl syntax OK
 ```
+
+# jenkins jobs
+There are manually maintained jobs and some jobs are now using `jenkins-job-builder` which defines jobs in yaml format.
```

Comment:
This should be a hyperlink to http://docs.openstack.org/infra/jenkins-job-builder/

Also nitpick: the line is too long ;-)